lib/components/user/auth0-token-wrapper.js

```diff
@@ -9,6 +9,7 @@ import AwaitingScreen from './awaiting-screen'
  * This component obtains and passes an Auth0 accessToken prop to its children.
  * It is neeeded because an Auth0 token is typically not available right away
  * when rendering for the first time.
+ * The component does nothing (besides adding a null accessToken prop) if no user is logged in.
```

Comment:
Since this is the case, let's merge the contents of this file into the auth0-wrapper.js file.

Reply:
Nevermind, this is needed because of the `componentDidMount` lifecycle method.